src/Runner.Worker/Handlers/CompositeActionHandler.cs

```diff
@@ -56,6 +56,9 @@ public async Task RunAsync(ActionRunStage stage)
                 childScopeName = $"__{Guid.NewGuid()}";
             }
 
+            // Set GITHUB_ACTION_PATH
+            ExecutionContext.SetGitHubContext("action_path", ActionDirectory);
```

Comment:
Since this set's the value globally in the `github` context. We need to make sure to clear it afterward

Reply:
oh shoot totally forgot. Will do.

Reply:
does this work when using job container?

Reply:
(I'll just set it for the step level github context)

Reply:
@TingluoHuang Sorry just realized my PR title was misleading. 

I was planning to support scripts for just Composite Run Steps as of now so I didn't explore if it would work for using job containers.

I think I'll explore that when we start building functionality for supporting "uses".

Right now, I haven't explored the "uses" for a container action. I'll write a note down and remember to explore this when I build functionality for that.

Thanks!

Reply:
(Will test after my Linux VM finishes)

Reply:
works for job containers: https://github.com/ethanchewy/testing-actions/runs/924499009?check_suite_focus=true